### PR TITLE
#17 make recipe list activity

### PR DESCRIPTION
**Pull request by SangwookYun**
Please check this code.

### CODE REVIEW

File: app/build.gradle [hunk 1]
```diff
@@ -25,8 +25,8 @@ android {
 
 dependencies {
     implementation fileTree(dir: 'libs', include: ['*.jar'])
-
     implementation 'androidx.appcompat:appcompat:1.1.0'
+    implementation 'com.google.android.material:material:1.1.0-beta01' //for floatingbutton
```
Comment (HaejoonChoi) [inline app/build.gradle:29]: 1.1.0 ??

File: app/src/main/java/ca/acsea/bakingappvan/MainActivity.java [hunk 2]
```diff
@@ -2,13 +2,53 @@
...
 
+import android.content.Intent;
 import android.os.Bundle;
+import android.view.View;
+import android.widget.Button;
+import android.widget.ImageView;
+import android.widget.TextView;
+import android.widget.Toast;
+
+import com.google.android.material.floatingactionbutton.FloatingActionButton;
```
Comment (HaejoonChoi) [inline app/src/main/java/ca/acsea/bakingappvan/MainActivity.java:13]: no need to import

File: app/src/main/res/values/colors.xml [hunk 3]
```diff
@@ -3,4 +3,7 @@
     <color name="colorPrimary">#6200EE</color>
     <color name="colorPrimaryDark">#3700B3</color>
     <color name="colorAccent">#03DAC5</color>
+    <color name="colorDark">#000000</color>
```
Comment (HaejoonChoi) [inline app/src/main/res/values/colors.xml:6]: No need

File: app/src/main/res/values/strings.xml [hunk 4]
```diff
@@ -1,3 +1,8 @@
 <resources>
     <string name="app_name">bakingAppVan</string>
+
+    <!--
+     The format of ID is "PageName_purpose"
+     -->
+    <string name="bakeEasy_recipeListActivity_title">Your Recipes</string>
```
Comment (HaejoonChoi) [inline app/src/main/res/values/strings.xml:7]: Add strings please

File: app/build.gradle (hunk 1 above)
Comment (SangwookYun) [reply app/build.gradle:29]: Remove -beta01. 
1.1.0 is a version Android recommends

File: app/src/main/res/values/colors.xml (hunk 3 above)
Comment (SangwookYun) [reply app/src/main/res/values/colors.xml:6]: I already applied this color on Activity. After "Layout Branch" is merged and I download on my local, I will remove this and change layout.

File: app/src/main/java/ca/acsea/bakingappvan/MainActivity.java (hunk 2 above)
Comment (SangwookYun) [reply app/src/main/java/ca/acsea/bakingappvan/MainActivity.java:13]: It is needed. Automatically imported.

File: app/src/main/res/values/strings.xml (hunk 4 above)
Comment (SangwookYun) [reply app/src/main/res/values/strings.xml:7]: Two string added overall.

**Comment by SangwookYun** [comment]
will

**Pull request merged**